RELEASE.md

```diff
@@ -6,6 +6,20 @@
 - [GNU Make](https://www.gnu.org/software/make/) `v4.x`
 - [Kustomize](https://github.com/kubernetes-sigs/kustomize) `v1.3.x`
 
+## Github Workflow Test Matrix Checkup
+
+**For all releases**
+
+We maintain some integration tests with 3rd party components which we need to manually verify and update before cutting any release.
+
+- [ ] check the testing workflow (`.github/workflows/test.yaml`) and ensure that all matrix versions are up to date for Kubernetes releases. If there have been any new releases (major, minor or patch) of Kubernetes since the latest version seen in that configuration make sure the new versions get added before proceeding with the release.
+- [ ] check the release workflow (`.github/workflows/release.yaml`) and ensure that all matrix versions are up to date for Kubernetes releases. If there have been any new releases (major, minor or patch) of Kubernetes since the latest version seen in that configuration make sure the new versions get added before proceeding with the release.
+- [ ] check the release workflow (`.github/workflows/release.yaml`) and ensure that all matrix versions are up to date for [Istio][istio] releases. If there have been any new releases (major, minor or patch) of Istio since the latest version seen in that configuration make sure the new versions get added before proceeding with the release.
+
+TODO: in the future we would like to automate these updates.
+
+[istio]:https://istio.io
+
```

Comment:
does this link really matter here? I'd be reasonably confident that anyone reading this file doesn't need this link. Or a link to kubernetes.io.

Reply:
I'm not really seeing that it's any problem to have these kinds of links, does it matter to me personally? Well no it doesn't but it's more just if someone is reading these docs and isn't really familiar with what Istio is, they can just click the link and go read.

Reply:
I suppose I'll just remove it, it's really minor.

```suggestion
```